watcher/watcher.go

```diff
@@ -88,6 +203,30 @@ func (wh *WatchHandler) ListImageIDsFromStorage() ([]string, error) {
 	return []string{}, nil
 }
 
+func (wh *WatchHandler) cleanUpMaps() {
+	wh.cleanUpImagesIDToWlidsMap()
+	wh.cleanUpWlidsToContainerToImageIDMap()
+}
+
+func (wh *WatchHandler) cleanUpImagesIDToWlidsMap() {
+	wh.imageIDsMapMutex.Lock()
+	defer wh.imageIDsMapMutex.Unlock()
+
+	wh.imagesIDToWlidsMap = make(map[string][]string)
+}
+
+func (wh *WatchHandler) cleanUpWlidsToContainerToImageIDMap() {
+	wh.wlidsToContainerToImageIDMapMutex.Lock()
+	defer wh.wlidsToContainerToImageIDMapMutex.Unlock()
+
+	wh.wlidsToContainerToImageIDMap = make(map[string]map[string]string)
```

Comment:
[TODO]: `map[string]map[string]string` **consider** defining a struct instead of having a map within a map.